main.py

```diff
@@ -40,7 +40,8 @@
 
 
 class ProblemRunner:
-    def __init__(self, problem="rproblem3", mode="feasibility", with_sdp=True, log_name=None):
+    def __init__(self, problem="rproblem3", mode="feasibility", with_sdp=True, log_name=None,
+                 save_sdp=False, save_esp=False):
```

Comment:
Is it better to set `True` to the default values of `save_sdp` and `save_esp`?

Reply:
My initial thought was to make sure that we never save the solutions by accident as this can negatively affect the runtime of the tests (especially with regards to ALNS). On second thought the effect is probably marginal, and if needed we can always just run `--save_sdp=False` or `--nosave_sdp`. Input @hkgrov ?

Reply:
Doesn't matter to much to me. I believe we should only save the best at the end of the run in both ALNS and ESP so it would matter very little. It shouldn't affect the runtime at all as iterations are done before write.

Reply:
Not as it stands now, but that was a really great idea! I will fix saving of SDP and ESP-results after ALSN-iterations :)